core/shared/src/main/scala/org/http4s/EntityEncoder.scala

```diff
@@ -122,7 +122,7 @@ object EntityEncoder {
     * bodies in advance.  As such, it does not calculate the Content-Length in
     * advance.  This is for use with chunked transfer encoding.
     */
-  implicit def streamEncoder[F[_], A](implicit
+  def streamEncoder[F[_], A](implicit
       W: EntityEncoder[F, A]
   ): EntityEncoder[F, Stream[F, A]] =
```

Comment:
This will make the change source-compatible. Users relying on the `implicit` will get a deprecation.

```suggestion
  @deprecated(...)
  implicit def implicitStreamEncoder[F[_], A](implicit
      W: EntityEncoder[F, A]
  ): EntityEncoder[F, Stream[F, A]] = streamEncoder[F, A]

  def streamEncoder[F[_], A](implicit
      W: EntityEncoder[F, A]
  ): EntityEncoder[F, Stream[F, A]] =
```